apps/kol-amelamdim/pages/login/index.tsx

```diff
@@ -70,6 +77,10 @@ const Login = () => {
             </Button>
           </Grid>
         </form>
+        <Typography component="h4">
+          עוד אין לכם משתמש ?{' '}
+          <Link onClick={() => router.push('/register')}>לחצו עלי להרשמה</Link>
+        </Typography>
```

Comment:
The Link should wrap the Typography. That way we can show the user it's clickable. right now It's shown with normal cursor and not pointer.